packager/signer/signer_test.go

```diff
@@ -609,6 +622,33 @@ func (this *SignerSuite) TestProxyTransformError() {
 	this.Assert().Equal(fakeBody, body, "incorrect body: %#v", resp)
 }
 
+func (this *SignerSuite) TestProxyHeadersUnaltered() {
+	urlSets := []util.URLSet{{
+		Sign: &util.URLPattern{[]string{"https"}, "", this.httpsHost(), stringPtr("/amp/.*"), []string{}, stringPtr(""), false, 2000, nil},
+	}}
+
+	// Generate a request for non-existent transformer that will fail
+	getTransformerRequest = func(r *rtv.RTVCache, s, u string) *rpb.Request {
+		return &rpb.Request{Html: string(s), DocumentUrl: u, Config: rpb.Request_CUSTOM,
+			AllowedFormats: []rpb.Request_HtmlFormat{rpb.Request_AMP},
+			Transformers:   []string{"bogus"}}
+	}
+	this.fakeHandler = func(resp http.ResponseWriter, req *http.Request) {
+		resp.Header().Set("Content-Type", "text/html")
+		resp.Header().Set("Set-Cookie", "chocolate chip")
+		resp.Header().Set("Cache-Control", "max-age=31536000")
+	}
+	resp := this.get(this.T(), this.new(urlSets), "/priv/doc?sign="+url.QueryEscape(this.httpsURL()+fakePath))
+	this.Assert().Equal(200, resp.StatusCode)
+
+	this.Assert().Equal("text/html", resp.Header.Get("content-type"))
```

Comment:
Is it possible to enumerate the full list here? e.g. [`this.Assert().ElementsMatch(...)`](https://godoc.org/github.com/stretchr/testify/assert#Assertions.ElementsMatch) or something. To prevent future changes from adding headers to the unsigned response.

Reply:
I took a stab at something like this in 2e0abcc . Let me know if it meets your expectations. Thanks again for the reviews!